be1-go/docs/README.md

```diff
@@ -68,13 +53,35 @@ logic in the hub module.
 
 #### Architecture
 
-The PoP Go backend expects actors to establish long lived websocket connections
-with it and send messages back and forth over websockets.
+The PoP Go backend expects actors (depending on the context these
+may be attendees/witnesses/organizer server) to establish long lived
+websocket connections with it and send messages back and forth over websockets
+using a publish/subscribe pattern.
 
 <div align="center">
   <img alt="Communication Stack" src="images/comm_stack.jpeg" width="600" />
 </div>
 
+On a higher level of abstraction, a client may publish messages or subscribe
+for messages on a *channel*. You may think of a channel as a topic which
+stores events that occur over time. For instance, every LAO is denoted by a unique
+channel (also called the root channel) and contains messages about all events
```

Comment:
isn't the root channel the one named `/root` , as opposed to LAO channels `/root/{lao-id}` ?

Reply:
Nice catch! Updated